plugins/net_plugin/net_plugin.cpp

```diff
@@ -684,28 +672,37 @@ namespace eosio {
    }
 
    void connection::txn_send_pending(const vector<transaction_id_type> &ids) {
-      for(auto t : my_impl->local_txns){
-         if(t.packed_transaction.size()) {
+      for(auto tx = my_impl->local_txns.begin(); tx != my_impl->local_txns.end(); ++tx ){
+         if(tx->packed_transaction.size()) {
             bool found = false;
             for(auto l : ids) {
-               if( l == t.id) {
+               if( l == tx->id) {
```

Comment:
The variable "l" looks way too much like a one in the github code font; we should probably use something a little more descriptive.